build/optaplanner-build-parent/pom.xml

```
@@ -30,7 +30,7 @@
     <version.com.fasterxml.jackson.core>2.12.3</version.com.fasterxml.jackson.core>
     <version.com.h2database>1.4.197</version.com.h2database>
     <version.com.thoughtworks.xstream>1.4.16</version.com.thoughtworks.xstream>
-    <version.io.quarkus>1.13.4.Final</version.io.quarkus>
+    <version.io.quarkus>2.0.0.CR2</version.io.quarkus>
```

```suggestion
    <version.io.quarkus>2.0.0.CR3</version.io.quarkus>
```

2.0.0.CR3 is available since a couple of hours